conf/server.yaml.template

```diff
@@ -8,12 +8,11 @@ SERVER:
     # RELEASE: populate with satellite version
     # The snap version currently testing (if applicable)
     # SNAP:
-    # The RHEL Base OS Version(x.y) where the Satellite is installed
-    # RHEL_VERSION:
     # The source of Satellite packages. Can be one of:
     # internal, ga, beta
     SOURCE: "internal"
-    RHEL_RELEASE: '7'
+    # The RHEL Base OS Version(x.y) where the Satellite is installed
+    RHEL_VERSION: '7'
```

Comment:
can you add this rhel_version setting to capsule.yaml.template as well ?

Reply:
> can you add this rhel_version setting to capsule.yaml.template as well
@Gauravtalreja1 Adding this will create inconsistency in CI and robottelo as "RHEL_RELEASE" is already there in CI for capsule.yaml. I will update it once we resolve that.

Reply:
@Gauravtalreja1 Updated.